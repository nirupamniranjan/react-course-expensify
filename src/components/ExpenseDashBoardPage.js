
 import React from 'react';

 import ExpenseList from './ExpenseList'
 import ExpenseListFilter from './ExpenseListFilter';
 import ExpensesSummary from './ExpenseSummary'
const ExpenseDashboardPage = () =>(


<div>
<ExpenseListFilter />
    <ExpenseList/>
    <ExpensesSummary/>
</div>

);


export default ExpenseDashboardPage;



