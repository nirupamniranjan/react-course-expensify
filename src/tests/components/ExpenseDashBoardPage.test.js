import React from 'react'
import ExpenseDashBoardPage from '../../components/ExpenseDashBoardPage'
import {shallow} from 'enzyme'


test('To Test Expense DashBoard Page',()=>{
    const wrapper = shallow(<ExpenseDashBoardPage/>)
    expect(wrapper).toMatchSnapshot();
})