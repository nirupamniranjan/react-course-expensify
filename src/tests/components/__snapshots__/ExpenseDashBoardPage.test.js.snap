// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To Test Expense DashBoard Page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
